dags/vz_populate_cr3_metadata.py

```diff
@@ -72,7 +70,7 @@ def get_env_vars():
         environment=env_vars,
         image="atddocker/vz-cr3-metadata-pdfs:production",
         auto_remove=True,
-        entrypoint=["/app/populate_cr3_file_metadata.py"],
+        entrypoint=["/app/populate_cr3_file_metadata.py -t 5 -a"],
```

Comment:
I may be reviewing this prematurely, but I went to check what -a means in the script, and I saw this comment: https://github.com/cityofaustin/atd-vz-data/pull/1436/files#r1566222905

should this include -v?

Reply:
I was just quoting your previous comment about airflow having -v and not -a!